src/Game/Client.java

```diff
@@ -1855,6 +1855,32 @@ private static String processClientCommand(String line) {
             Settings.takingSceneryScreenshots = false;
             break;
           }
+        case "systemfonts":
+          {
+            if (GameApplet.loadSystemFonts()) {
+              displayMessage("Loaded system fonts", CHAT_QUEST);
+            } else {
+              displayMessage("Unable to load system fonts...", CHAT_QUEST);
+            }
+          }
+          break;
+        case "jffonts":
+        {
+          GameApplet.loadJfFonts();
+          displayMessage("Loaded jf fonts", CHAT_QUEST);
+          break;
+        }
```

Comment:
Additionally, these are separate from the options toggles, so they get out of sync with those settings if you invoke them... Does it matter here?